code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -807,3 +808,184 @@
+	..()
+	if(world.time > last_gong_time + gong_cooldown)
+		gong(H)
+
+/datum/species/golem/bronze/on_hit(obj/item/projectile/P, mob/living/carbon/human/H)
+	..()
+	if(world.time > last_gong_time + gong_cooldown)
+		gong(H)
+
+/datum/species/golem/bronze/proc/gong(mob/living/carbon/human/H)
+	last_gong_time = world.time
+	for(var/mob/living/M in get_hearers_in_view(7,H))
+		if(M.stat == DEAD)	//F
+			return
+		if(M == H)
+			H.show_message("<span class='narsiesmall'>You cringe with pain as your body rings around you!</span>", 2)
+			H.playsound_local(H, 'sound/effects/gong.ogg', 100, TRUE)
+			H.soundbang_act(2, 0, 100, 1)
+			H.jitteriness += 7
+		var/distance = max(0,get_dist(get_turf(H),get_turf(M)))	
+		switch(distance)
+			if(0 to 1)
+				M.show_message("<span class='narsiesmall'>GONG!</span>", 2)
+				M.playsound_local(H, 'sound/effects/gong.ogg', 100, TRUE)
+				M.soundbang_act(1, 0, 30, 3)
+				M.confused += 10
+				M.jitteriness += 4
+			if(2 to 3)
+				M.show_message("<span class='cult'>GONG!</span>", 2)
+				M.playsound_local(H, 'sound/effects/gong.ogg', 75, TRUE)
+				M.soundbang_act(1, 0, 15, 2)
+				M.jitteriness += 3
+			else
+				M.show_message("<span class='warning'>GONG!</span>", 2)
+				M.playsound_local(H, 'sound/effects/gong.ogg', 50, TRUE)
+
+
+/datum/species/golem/cardboard //Faster but weaker, can also make new shells on its own
+	name = "Cardboard Golem" 
+	id = "cardboard golem"
+	prefix = "Cardboard"
+	special_names = null
+	info_text = "As a <span class='danger'>Cardboard Golem</span>, you aren't very strong, but you are a bit quicker and can easily create more brethren by using cardboard on yourself."
+	species_traits = list(NOBLOOD,NO_UNDERWEAR,NOEYES)
+	inherent_traits = list(TRAIT_NOBREATH, TRAIT_RESISTCOLD,TRAIT_RESISTHIGHPRESSURE,TRAIT_RESISTLOWPRESSURE,TRAIT_NOGUNS,TRAIT_RADIMMUNE,TRAIT_PIERCEIMMUNE,TRAIT_NODISMEMBER)
+	limbs_id = "c_golem" //special sprites
+	attack_verb = "whips"
+	attack_sound = 'sound/weapons/whip.ogg'
+	miss_sound = 'sound/weapons/etherealmiss.ogg'
+	fixed_mut_color = null
+	armor = 25
+	burnmod = 1.25
+	heatmod = 2
+	speedmod = 1.5
+	punchdamagelow = 4
+	punchstunthreshold = 7
+	punchdamagehigh = 8
+	var/last_creation
+	var/brother_creation_cooldown = 300
+
+/datum/species/golem/cardboard/spec_attacked_by(obj/item/I, mob/living/user, obj/item/bodypart/affecting, intent, mob/living/carbon/human/H)
+	. = ..()
+	if(user != H || intent != INTENT_HELP)
+		return FALSE //forced reproduction is rape.
+	if(istype(I, /obj/item/stack/sheet/cardboard))
+		var/obj/item/stack/sheet/cardboard/C = I
+		if(last_creation + brother_creation_cooldown < world.time) //no cheesing dork
+			return
+		to_chat(H, "<span class='notice'>You attept to create a new cardboard brother.</span>")
+		if(do_after(user, 30, target = user))
+			if(last_creation + brother_creation_cooldown < world.time) //no cheesing dork
+				return
+			if(!C.use(10))

```

Comment:
It would also might cause some edge cases where you had enough before but during the do_after you lost some, but i'm not a fan of having the timer go through only to say lolnope though.

`if(C.amount >= 10)` preliminary check maybe?

Reply:
sure I'll add that